app/components/locale-selector.js

```diff
@@ -10,7 +10,7 @@ export default class LocaleSelector extends Component {
     this.locales = [
       {text: 'English', value: 'en'},
       {text: '日本語', value: 'ja'},
-      {text: '한국어', value: 'kr'},
+      {text: '한국어', value: 'ko'},
```

Comment:
One thing it complained about was saying "kr" was a non-standard language value, so i just google "korean language value" and google told me to use "ko" so i did.

Reply:
Ah ok, I didn't notice that before. Not sure if that means we have to change the filename of the locale file too?

Reply:
I did that :)

Reply:
Ah thanks, my bad didn't notice. :+1:

Reply:
![localepicker](https://user-images.githubusercontent.com/66243/84216976-cfacd380-aa7f-11ea-8853-06f9ef1f84ef.gif)

The locale picker is acting a bit strange, it doesn't seem to update the selected value when switching, it still says "English".

Reply:
Interestingly, if you click the same language twice, it changes correctly. I think this has to do with the "fallback language" thing from `ember-intl`. I'll investigate further...

Reply:
Ok, I found the issue. Should be fixed in my latest commit. @mcfiredrill